Add new Buildbot worker: Link to instructions for BuildBot admins (#1848)
python/devguide: testing/new-buildbot-worker.rst

In order to set up the buildbot software, you will need to obtain an identifier
and password for your worker so it can join the fleet.  Open an issue in the
`configuration repository <https://github.com/python/buildmaster-config/issues/new?template=new_worker.yml>`__

> to discuss adding your worker and to obtain the needed workername and password.
> (Instructions for Buildbot administrators are `in the configuration README <https://github.com/python/buildmaster-config#add-a-worker>`__.)
> 
> You can do some of the steps that follow

Author: ```suggestion
to discuss adding your worker and to obtain the needed workername and password.
(The steps the administrators follow are documented under `"Add a worker"
<https://github.com/python/buildmaster-config#add-a-worker>`__ in the README.)
You can do some of the steps that follow
```
